### DISCUSSION TITLE
Multiple UPS'

### DISCUSSION THREAD

**Discussion by wx4cb** [nutdotnet/Coco.Nut#7] [Ideas]
The ability to check multiple ups with one instance of the program.

This could simply be as a drop down to select which one you want to look at or even "auto populate" or "grow" the current window to the number of ups' the NUT instance returns

Finding out is REALLY easy as there is a "LIST UPS" command that you can use to find all connected ups' attached to the daemon. As a matter of course, you should actually do that anyway to make sure that the ups requested (currently doesnt appear to be any checks on the ups name) is actually present and valid.

doing it this way would mean there is ZERO ambiguity as to which UPS are available.

This could then just populate a drop down and a global variable could be used to say "this is the one we're using" for any commands and all the list box has to do is just change that global and the rest of the program pulls in the data automatically on the next refresh from the daemon.

**Comment by rederikus** [comment]
Yes, me too.  Thanks.

Right now I only have two UPS devices.  One on each floor of my house but this will probably expand of time.  A drop down list of devices would be a good way to implement access.
